client/scripts/views/modals/new_topic_modal.ts

```diff
@@ -4,30 +4,32 @@ import m from 'mithril';
 import app from 'state';
 import $ from 'jquery';
 import { Button, Input, Form, FormGroup, FormLabel } from 'construct-ui';
-
+import BN from 'bn.js';
 import { confirmationModalWithText } from 'views/modals/confirm_modal';
 import { CompactModalExitButton } from 'views/modal';
-
+import { expandTokenAmount } from 'helpers';
 interface INewTopicModalForm {
   id: number,
   name: string,
   description: string,
+  token_threshold: string
 }
 
 const NewTopicModal: m.Component<{
   id: number,
   name: string,
   description: string,
+  token_threshold: string
```

Comment:
I feel like `token_threshold` properties should be optional? Like it only matters in Token/ETH-Governance communities